src/lightning/data/streaming/cache.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         chunk_size: Optional[int] = None,
         chunk_bytes: Optional[Union[int, str]] = None,
         item_loader: Optional[BaseItemLoader] = None,
-        max_cache_size: Union[int, str] = "200GB",
+        max_cache_size: Optional[Union[int, str]] = None,
```

Comment:
Can you add in the argument docstring how we determine the size here by default?

Reply:
I changed it to 100GB.

Reply:
I still see it as None, did you forget to push the change?